BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -137,23 +140,25 @@ fun FavoriteCollectionCard(
     modifier: Modifier = Modifier
 ) {
     Surface(
-        shape = MaterialTheme.shapes.small,
+        shape = MaterialTheme.shapes.medium,
+        color = MaterialTheme.colorScheme.surfaceVariant,
         modifier = modifier
     ) {
         Row(
             verticalAlignment = Alignment.CenterVertically,
-            modifier = Modifier.width(192.dp)
+            modifier = modifier.width(255.dp)
         ) {
             Image(
                 painter = painterResource(drawable),
                 contentDescription = null,
-                contentScale = ContentScale.Crop,
-                modifier = Modifier.size(56.dp)
+                contentScale = ContentScale.FillBounds,
```

Comment:
Is `FillBounds` needed? Can we keep this to `Crop`?

Reply:
Yeah - we can change back to Crop